src/WebJobs.Script.Grpc/WorkerInvocationMetrics.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    public class WorkerInvocationMetrics
```

Comment:
Should this type have `public` visibility? Would `internal` work?

Reply:
Done